tests/acceptance/blueprint-test.js

```diff
@@ -57,4 +58,67 @@ describe('Blueprint', function() {
         expect(actual).to.equal('// A test for foo\n');
       });
   });
+
+  it('respects editorconfig', function() {
+    var exampleBlueprint = Blueprint.load('tests-fixtures/blueprints/example-web-app');
+
+    var options = {
+      entity: {
+        name: 'foo'
+      },
+      ui,
+      target: tmpdir,
+      project: {
+        config: function() {
+          return {};
+        },
+        name: function() {
+          return 'foo';
+        },
+        root: tmpdir,
+        // TODO: refactor out
+        isEmberCliProject: function() {
+          return true;
+        },
+        // TODO: refactor out
+        isEmberCLIAddon: function() {
+          return true;
+        }
+      },
+    };
+
+    fixturify.writeSync(tmpdir, {
+      '.editorconfig': `
+root = true
+
+[*]
+indent_style = space
+indent_size = 1
+end_of_line = crlf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+`,
+      'app': {
+        '.editorconfig': `
+[*]
+indent_size = 3
+insert_final_newline = false
+`
+      }
+    })
+
+    return exampleBlueprint.install(options)
+      .then(function() {
+        var actual = fixturify.readSync(tmpdir);
+
+        // check project root settings
+        expect(actual.tests['foo-test.js']).to.equal(`if (1) {\r\n // originally poor formatted file\r\n}\r\n`);
+
+        // check folder specific settings and some of filetypes
+        expect(actual.app['foo.js']).to.equal(`if (1) {\r\n   // originally poor formatted file\r\n}`);
+        expect(actual.app['foo.css']).to.equal(`.if {\r\n   color: aquamarine;\r\n}`);
+        // expect(actual.app['foo.hbs']).to.equal(`{{#if}}\r\n   originally poor formatted\r\n{{/if}}`);
```

Comment:
the best I could achieve at this moment is handlebars template collapsed in a single line